### DISCUSSION TITLE
Bulk SMS with Notification and Job support

### DISCUSSION THREAD

**Discussion by saaiful** [HassanZahirnia/laravel-package-ocean#36] [Package Suggestions]
https://packagist.org/packages/bluedot-bd/laravel-bulksms

**Comment by divyaniverma** [comment]
MsgClub Solutions is a leading and reputable bulk SMS service provider . To distribute bulk SMS, we use reliable SMS gateways that are implemented and maintained on our own servers. We help users set up their accounts using our gateway.Bulk SMS Also helpful to share notification and job support. Bulk SMS help to sharing job profile.
